source/content/guides/jenkins.md

```diff
@@ -11,6 +11,7 @@ newcms: [drupal, wordpress]
 audience: [development]
```

Comment:
@stovak , contains multiple references to D8. Should it say “prior version” instead?  See preview link: https://pr-8004--pantheon-docs.my.pantheonfrontend.website/docs/guides/jenkins